packages/dd-trace/test/appsec/index.multer.plugin.spec.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const { channel } = require('dc-polyfill')
+const axios = require('axios')
+const path = require('path')
+const agent = require('../plugins/agent')
+const appsec = require('../../src/appsec')
+const Config = require('../../src/config')
+const { json } = require('../../src/appsec/blocked_templates')
+
+const multerReadCh = channel('datadog:multer:read:finish')
+
+withVersions('multer', 'multer', version => {
```

Comment:
same here:
why is this tests here, and not as integration test using sandbox to test with the specific multer version?

Reply:
this one is the original, to test multipart request blocking but now with the [integration](https://github.com/DataDog/dd-trace-js/pull/4781/files#diff-0f99f954cb822f73a211aebfe34a4d7ad9305eeabc5f0c864e5162ca923566b8) test is kind of duplicated

Reply:
deleted